### DISCUSSION TITLE
Conti di saldo unauthorized

### DISCUSSION THREAD

**Discussion by Mingo07-dev** [fattureincloud/api#432] [Q&A] [answered]
Hi, 
One of our client, when trying to make a payment relative to an invoice, can't see the list of conti di saldo. I tried doing the same and it throws this error: 
Exception when calling SuppliersApi->listSuppliers: [401] Client error: `GET https://api-v2.fattureincloud.it/c/******/info/payment_accounts?fields=id%2Cname` resulted in a `401 Unauthorized` response: {"error":{"message":"Unauthorized."}} null

The company id is correct as they can create an invoice and more. 

This is the code:

```
<?php
session_start();
if($_SESSION['debug'] == true){
	error_reporting(E_ALL);
} else{
	error_reporting(E_ALL);
	ini_set('display_errors', 'Off');
	ini_set("log_errors", 1);
}
include ('connect.php');
 
    //SESSIONE
    if ( isset($_SESSION["user_id"]) ){ 
        $emitter = $_SESSION["user_id"];
    } else { header("location:logout.php?code=4"); }

    $company_id = $_POST['FIC_COMPANY_ID'];
    
    require_once('../vendor/autoload.php');

    $token = $_SESSION['FiCtoken'];
    $config = FattureInCloud\Configuration::getDefaultConfiguration()->setAccessToken($token);
    $apiInstance = new FattureInCloud\Api\InfoApi(new GuzzleHttp\Client(), $config);

    $id = 0;
    $name = '';

    try {
        $result = $apiInstance->listPaymentAccounts($company_id,"id,name");
        $res = json_decode($result, true);
        
    } catch (Exception $e) {
        echo 'Exception when calling SuppliersApi->listSuppliers: ', $e->getMessage(), PHP_EOL;
    }
    // echo($result);
    echo(json_encode($res));


```

Could be a problem with the permissions on settings-> applicazioni collegate?
I checked and the settings permissions are on 'complete access'

**Comment by valmoz** [comment]
Hi @Mingo07-dev 
first, you should NEVER post the company ID of the user on this community, since this is a public post.
For this kind of issue that requires sharing sensitive data, and to be able to answer as soon as possible, you should write to our [Customer Support](https://developers.fattureincloud.it/docs/support).

That said, we checked and noticed that this user is having some 200 OK answers on this method, can you confirm that this was solved?

Lastly, the log is wrong, since this is not the listSuppliers method but the listPaymentAccounts one, better to fix it because it could be misleading :D

Thanks
Mauro

**Reply by Mingo07-dev** [reply, marked as answer]
Hi @valmoz, 
thank you for pointing that out to me. I already removed it.
By the way as you said, it is resolved, just some minutes after it started working again.
Thank you
